ui/lib/utils/formatSql.ts

```diff
@@ -1,5 +1,7 @@
-import sqlFormatter from 'sql-formatter-plus-plus'
+import MySqlFormatter from './MySqlFormatter/MySqlFormatter'
```

Comment:
Maybe better to arrange formatSql and MySqlFormatter in the same directory since they are deeply linked. Also maybe we can name it TiDBSQLFormatter :)

Reply:
Ok, nice naming!